components/shared/side-navbar/SideNavbar.tsx

```diff
@@ -28,12 +28,8 @@ const SideNavbar = () => {
           <div className='space-y-4 pb-12'>
             <Link href='/posts/create-post'>
               <button className="paragraph-4-medium primary-gradient flex w-full flex-row items-center justify-center rounded border border-none py-3 font-bold text-white">
-                <Image
-                src={plusCircleIcon}
-                alt="Plus Icon"
-                className='mr-2'
-                />
-                Create Post
+                <PlusIcon />
+                <span className='ml-2'>Create Post</span>
```

Comment:
Same adjustments here for the text color, unneeded flex-row, py-2 adjustment, and suggestion to use `gap-2` without a span.